frontend/components/forms/Input.tsx

```diff
@@ -1,24 +1,32 @@
+import React from 'react'
+
 interface InputProps extends React.InputHTMLAttributes<HTMLInputElement> {
   label: string
+  isError?: boolean
 }
 
-export const Input: React.FC<InputProps> = (props) => {
-  const { label, ...rest } = props
+export const Input: React.FC<InputProps> = React.forwardRef<any, InputProps>(
```

Comment:
Why does it need to be `React.forwardRef`? And I see an `any` 😄

Reply:
Since using `react-hook-form`, `register` method returns a `ref`. So in order to pass the `ref`, need to use `forwardRef`.


And for the `any`... hahahaha. 😅